### PR TITLE
8067 zdb should be able to dump literal embedded block pointer

### PR DESCRIPTION
**Pull request by ahrens**
Reviewed by: George Wilson <george.wilson@delphix.com>
Reviewed by: Alex Reece <alex@delphix.com>

Upstream bugs: DLPX-33322

### CODE REVIEW

**Comment by ahrens** [comment]
@zettabot go

File: usr/src/cmd/zdb/zdb.c [hunk 1]
```diff
@@ -150,6 +152,8 @@ usage(void)
 	(void) fprintf(stderr, "        -C config (or cachefile if alone)\n");
 	(void) fprintf(stderr, "        -d dataset(s)\n");
 	(void) fprintf(stderr, "        -D dedup statistics\n");
+	(void) fprintf(stderr, "        -E decode and display block from an "
+	    "embedded block pointer\n\n");
```
Comment (yuripv) [inline usr/src/cmd/zdb/zdb.c]: just wondering why double ```\n``` here?

**Comment by yuripv** [comment]
This seems to be missing man page updates, do you need help with it?

File: usr/src/cmd/zdb/zdb.c (hunk 1 above)
Comment (ahrens) [inline usr/src/cmd/zdb/zdb.c]: I think this is a mistake, thanks for catching it.

**Comment by ahrens** [comment]
> This seems to be missing man page updates, do you need help with it?

 @yuripv You're right.  It's my responsibility, but I'd gladly accept the help if you'd like to update the manpage!

**Comment by ahrens** [comment]
@zettabot go

**Comment by yuripv** [comment]
How about the following, feel free to improve the description text:
```
@@ -10,10 +10,10 @@
 .\"
 .\"
 .\" Copyright 2012, Richard Lowe.
-.\" Copyright (c) 2012, 2016 by Delphix. All rights reserved.
+.\" Copyright (c) 2012, 2017 by Delphix. All rights reserved.
 .\" Copyright 2017 Nexenta Systems, Inc.
 .\"
-.Dd January 14, 2017
+.Dd April 14, 2017
 .Dt ZDB 1M
 .Os
 .Sh NAME
@@ -39,6 +39,10 @@
 .Op Fl A
 .Op Fl U Ar cache
 .Nm
+.Fl E
+.Op Fl A
+.Ar word0 Ns : Ns Ar word1 Ns :...: Ns Ar word15
+.Nm
 .Fl l
 .Op Fl Aqu
 .Ar device
@@ -153,6 +157,10 @@
 Dump the contents of the deduplication tables describing duplicate blocks.
 .It Fl DDDDD
 Also dump the contents of the deduplication tables describing unique blocks.
+.It Fl E Ar word0 Ns : Ns Ar word1 Ns :...: Ns Ar word15
+Decode and display block from an embedded block pointer specified by the
+.Ar word
+arguments.
 .It Fl h
 Display pool history similar to
 .Nm zpool Cm history ,
```

**Comment by ahrens** [comment]
@yuripv Awesome, thank you!

**Comment by ahrens** [comment]
@zettabot go

**Review by ghost — APPROVED** [review]

**Pull request closed without merging**